things/thing-4.md

```diff
@@ -0,0 +1,147 @@
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+#Thing 4: Data Discovery
+
+**[&lt;&lt;Jump to Thing 3](thing-3.md) &nbsp; &nbsp; [Jump to Thing 5&gt;&gt;](thing-5.md)**
+
+Repositories and portals play an important role in making research data
+discoverable and accessible.
+
+-   **Getting started:** explore Research Data Australia to find
+    research data
+-   **Learn more:** what's in (and what's not in) the 2,000+ data
+    repositories in re3data?
+-   **Challenge me:** can we Trust that repository and how would we
+    know?
+
+
+* [Getting started](#getting-started)
+* [Learn more](#learn-more)
+* [Challenge me](#challenge-me)
+
+## Getting started 
+
+### Repositories for data discovery
+
+Repositories enable discovery of data by publishing data descriptions
+("metadata") about the data they hold - like a library catalogue
+describes the materials held in a library.  Most repositories provide
+access to the data itself, but not always.
+
+Data portals or aggregators draw together research data records from a
+number of repositories. eg Research Data Australia (RDA) aggregates
+records from over 100 Australian repositories.
+
+1.  Click on this RDA record from the Australian Antarctic Data Centre:
+    [Weddell seals in
+    Antarctica](https://researchdata.ands.org.au/measuring-effects-human-leptonychotes-weddellii/699460 "Weddell Seals")
+2.  Have a close look at the record to see the ways the Australian
+    Antarctic Division has made this record discoverable and
+    accessible.  Note how many times this dataset has been cited and how
+    to cite this data.  We will look at data citation in more detail in
+    Thing 7.
+3.  Spend a few minutes exploring RDA.
+    1.  Try
+        [browsing](https://researchdata.ands.org.au/subjects "Browse subjects in Research Data Australia")
+        or searching on a topic of interest.
+    2.  See [which
+        institutions](https://researchdata.ands.org.au/contributors "Research Data Australia - contributors")
+        contribute metadata records to RDA.
+    3.  Explore a record or two in depth.
+
+**Consider:** the future impact of having a national research data
+catalogue.
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag#23things
+
+Go to [Thing 5](thing-5.md)
+Data sharing or to [All Things](index.md)
+
+## Learn More 
+
+### Finding data repositories
+
+What data repositories exist and how are Australian researchers sharing
+their data?
+
+-   Start by going to [re3data.org](http://www.re3data.org/)
+-   Click on Browse &gt; By Country &gt; click on Australia in the map
+-   There are 87 repositories listed for Australia. Does this represent
+    all the research data repositories Australia has to offer? Is
+    anything missing?
+
```

Comment:
Today there are 90 repositories listed.  Suggest an alteration of language to avoid the figure going out of date.  One approach (assuming that the number won't fall), could instead state "There are at least 90 repositories...".  Another approach would be to rework to ask a question like "How many repositories are listed for Australia. Does this represent..."

Reply:
Updated to "How many repositories are listed for Australia?"